integration-tests/test/fee-payment.spec.ts

```diff
@@ -152,12 +157,17 @@ describe('Fee Payment Integration Tests', async () => {
   })
 
   it('should be able to withdraw fees back to L1 once the minimum is met', async function () {
+    if (isLiveNetwork()) {
+      this.skip()
+      return
+    }
+
     const l1FeeWallet = await env.sequencerFeeVault.l1FeeWallet()
     const balanceBefore = await env.l1Wallet.provider.getBalance(l1FeeWallet)
     const withdrawalAmount = await env.sequencerFeeVault.MIN_WITHDRAWAL_AMOUNT()
 
     const l2WalletBalance = await env.l2Wallet.getBalance()
-    if (IS_LIVE_NETWORK && l2WalletBalance.lt(withdrawalAmount)) {
+    if (l2WalletBalance.lt(withdrawalAmount)) {
```

Comment:
This condition should only be present when `IS_LIVE_NETWORK === true`, so we should be OK to remove this block entirely. If testing locally, the wallet should always have enough funds. If the wallet does not have enough funds, the test should throw an error (correctly) instead of exiting cleanly like it does in this block.